Open N5 dialogue broken

The dialogue to open N5 files seems to be broken:
When I press `ctrl` + `o` and then select `N5`, I get a long error message (see below).
The dialogue opens, but I can't open any files.
Note that opening HDF5 files works fine.

I use the latest conda version of paintera (0.7.0).

```
(paintera) pape@vm-kreshuk-01:~/Work/my_projects/paintera_projects$ ./run_paintera.sh rachel_parapodium
Nov 08, 2018 10:14:58 AM javafx.beans.binding.Bindings$6 computeValue
WARNING: Exception while evaluating binding
java.lang.RuntimeException: java.nio.file.FileAlreadyExistsException: /home/pape
	at org.janelia.saalfeldlab.util.MakeUnchecked.lambda$supplier$5(MakeUnchecked.java:111)
	at org.janelia.saalfeldlab.util.MakeUnchecked.lambda$supplier$6(MakeUnchecked.java:122)
	at java.util.Optional.map(Optional.java:215)
	at org.janelia.saalfeldlab.paintera.ui.opendialog.menu.n5.GenericBackendDialogN5.lambda$new$1(GenericBackendDialogN5.java:107)
	at javafx.beans.binding.Bindings$6.computeValue(Bindings.java:338)
	at javafx.beans.binding.ObjectBinding.get(ObjectBinding.java:153)
	at javafx.beans.binding.Bindings$67.computeValue(Bindings.java:5935)
	at javafx.beans.binding.BooleanBinding.get(BooleanBinding.java:157)
	at javafx.beans.binding.Bindings$52.computeValue(Bindings.java:4691)
	at javafx.beans.binding.BooleanBinding.get(BooleanBinding.java:157)
	at javafx.beans.property.BooleanPropertyBase.get(BooleanPropertyBase.java:130)
	at javafx.scene.Node$MiscProperties.isDisable(Node.java:6704)
	at javafx.scene.Node.isDisable(Node.java:1569)
	at javafx.scene.Node.updateDisabled(Node.java:1679)
	at javafx.scene.Node.access$500(Node.java:384)
	at javafx.scene.Node$MiscProperties$8.invalidated(Node.java:6712)
	at javafx.beans.property.BooleanPropertyBase.markInvalid(BooleanPropertyBase.java:109)
	at javafx.beans.property.BooleanPropertyBase.access$000(BooleanPropertyBase.java:49)
	at javafx.beans.property.BooleanPropertyBase$Listener.invalidated(BooleanPropertyBase.java:245)
	at com.sun.javafx.binding.ExpressionHelper$SingleInvalidation.fireValueChangedEvent(ExpressionHelper.java:137)
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:81)
	at javafx.beans.binding.BooleanBinding.invalidate(BooleanBinding.java:176)
	at com.sun.javafx.binding.BindingHelperObserver.invalidated(BindingHelperObserver.java:51)
	at com.sun.javafx.binding.ExpressionHelper$Generic.fireValueChangedEvent(ExpressionHelper.java:349)
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:81)
	at javafx.beans.binding.BooleanBinding.invalidate(BooleanBinding.java:176)
	at com.sun.javafx.binding.BindingHelperObserver.invalidated(BindingHelperObserver.java:51)
	at com.sun.javafx.binding.ExpressionHelper$Generic.fireValueChangedEvent(ExpressionHelper.java:349)
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:81)
	at javafx.beans.binding.ObjectBinding.invalidate(ObjectBinding.java:172)
	at com.sun.javafx.binding.BindingHelperObserver.invalidated(BindingHelperObserver.java:51)
	at com.sun.javafx.binding.ExpressionHelper$SingleInvalidation.fireValueChangedEvent(ExpressionHelper.java:137)
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:81)
	at javafx.beans.property.ObjectPropertyBase.fireValueChangedEvent(ObjectPropertyBase.java:105)
	at javafx.beans.property.ObjectPropertyBase.markInvalid(ObjectPropertyBase.java:112)
	at javafx.beans.property.ObjectPropertyBase.access$000(ObjectPropertyBase.java:51)
	at javafx.beans.property.ObjectPropertyBase$Listener.invalidated(ObjectPropertyBase.java:233)
	at com.sun.javafx.binding.ExpressionHelper$SingleInvalidation.fireValueChangedEvent(ExpressionHelper.java:137)
	at com.sun.javafx.binding.ExpressionHelper.fireValueChangedEvent(ExpressionHelper.java:81)
	at javafx.beans.property.ObjectPropertyBase.fireValueChangedEvent(ObjectPropertyBase.java:105)
	at javafx.beans.property.ObjectPropertyBase.markInvalid(ObjectPropertyBase.java:112)
	at javafx.beans.property.ObjectPropertyBase.set(ObjectPropertyBase.java:146)
	at org.janelia.saalfeldlab.paintera.ui.opendialog.menu.n5.FileSystem.backendDialog(FileSystem.java:71)
	at org.janelia.saalfeldlab.paintera.ui.opendialog.menu.n5.N5OpenSourceDialog$N5FSOpener.lambda$onAction$0(N5OpenSourceDialog.java:85)
	at org.janelia.saalfeldlab.paintera.ui.opendialog.menu.OpenDialogMenu.lambda$getContextMenu$0(OpenDialogMenu.java:56)
	at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
	at com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:49)
	at javafx.event.Event.fireEvent(Event.java:198)
	at javafx.scene.control.MenuItem.fire(MenuItem.java:462)
	at com.sun.javafx.scene.control.skin.ContextMenuContent$MenuItemContainer.doSelect(ContextMenuContent.java:1405)
	at com.sun.javafx.scene.control.skin.ContextMenuContent$MenuItemContainer.lambda$createChildren$343(ContextMenuContent.java:1358)
	at com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent(CompositeEventHandler.java:218)
	at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:80)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
	at com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
	at com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:54)
	at javafx.event.Event.fireEvent(Event.java:198)
	at javafx.scene.Scene$MouseHandler.process(Scene.java:3757)
	at javafx.scene.Scene$MouseHandler.access$1500(Scene.java:3485)
	at javafx.scene.Scene.impl_processMouseEvent(Scene.java:1762)
	at javafx.scene.Scene$ScenePeerListener.mouseEvent(Scene.java:2494)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:394)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$353(GlassViewEventHandler.java:432)
	at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:389)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:431)
	at com.sun.glass.ui.View.handleMouseEvent(View.java:555)
	at com.sun.glass.ui.View.notifyMouse(View.java:937)
	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.lambda$null$48(GtkApplication.java:139)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.nio.file.FileAlreadyExistsException: /home/pape
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:88)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:384)
	at java.nio.file.Files.createDirectory(Files.java:674)
	at java.nio.file.Files.createAndCheckIsDirectory(Files.java:781)
	at java.nio.file.Files.createDirectories(Files.java:727)
	at org.janelia.saalfeldlab.n5.N5FSWriter.<init>(N5FSWriter.java:69)
	at org.janelia.saalfeldlab.n5.N5FSWriter.<init>(N5FSWriter.java:92)
	at org.janelia.saalfeldlab.paintera.ui.opendialog.menu.n5.FileSystem.lambda$backendDialog$4(FileSystem.java:71)
	at org.janelia.saalfeldlab.util.MakeUnchecked.lambda$supplier$6(MakeUnchecked.java:119)
	... 85 more
```

--- a/src/main/java/org/janelia/saalfeldlab/paintera/ui/opendialog/menu/n5/FileSystem.java
+++ b/src/main/java/org/janelia/saalfeldlab/paintera/ui/opendialog/menu/n5/FileSystem.java
@@ -16,6 +16,7 @@ import org.slf4j.LoggerFactory;
 
 import java.io.File;
 import java.lang.invoke.MethodHandles;
+import java.nio.file.Paths;
 import java.util.Optional;
 import java.util.concurrent.ExecutorService;
 import java.util.function.Consumer;
@@ -28,7 +29,8 @@ public class FileSystem {
 
 	private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-	private final StringProperty container = new SimpleStringProperty(USER_HOME);
+	private final StringProperty container = new SimpleStringProperty(MakeUnchecked.supplier(Paths.get(USER_HOME)::toRealPath).get().toString());
+
 	private final ObjectProperty<Supplier<N5Writer>> writerSupplier = new SimpleObjectProperty<>(MakeUnchecked.supplier(() -> null));
 
 	{

